java/src/jmri/jmrix/can/cbus/swing/nodeconfig/CbusAllocateNodeNumber.java

```diff
@@ -301,8 +317,9 @@ private void processNodeName(CanReply m){
             nodepropbuilder.append (CbusNodeConstants.getManu( _paramsArr[0] ));  
             nodepropbuilder.append (" ");
             nodepropbuilder.append (_tempNodeName);
-            
-            rqNNtext.setText(nodepropbuilder.toString());
+            ThreadingUtil.runOnGUI( ()->{
```

Comment:
I would think if anything you want to run this on the LayoutThread, not the Gui thread (these may currently be the same, but they don't have to be long term...)

Reply:
rqNNtext is a javax.swing.JLabel being displayed within a javax.swing.JOptionPane .
Updating the JLabel updates the GUI.